### DISCUSSION TITLE
Please don't remove the "Trending" label, it is very useful

### DISCUSSION THREAD

**Discussion by dyggod** [community/community#31720] [Other Feature Feedback, Questions, & Ideas] [answered]
As a Chinese developer, "Trending" is an important tool for me to reach the vast developer world.
It is an important driving force for me to browse github every day.

So please keep this tag, and if can't keep it, please post an alternate feature.

**Comment by pmg1991** [comment]
Please keep "Trending Repositories and Trending Developers

**Comment by dyggod** [comment]
This may require more people to speak up.

**Comment by dyggod** [comment, marked as answer]
Glad we saw feedback from the community：#31644
